crates/uv-resolver/src/pubgrub/report.rs

```diff
@@ -357,7 +357,21 @@ impl PubGrubReportFormatter<'_> {
                 External::NoVersions(package, set, _) => {
                     // Check for no versions due to pre-release options
                     if let Some(selector) = selector {
-                        if set.bounds().any(Version::any_prerelease) {
+                        let any_prerelease = set.iter().any(|(start, end)| {
+                            let is_pre1 = match start {
+                                Bound::Included(v) => v.any_prerelease(),
```

Comment:
Nit: can we use `version` or `segment` instead of `v`?

Reply:
good call